types/Webhooks.d.ts

```diff
@@ -36,7 +36,12 @@ declare module 'stripe' {
         /**
          * Optional CryptoProvider to use for computing HMAC signatures.
          */
-        cryptoProvider?: CryptoProvider
+        cryptoProvider?: CryptoProvider,
+
+        /**
+         * Optional: timestamp to use when checking signature validity. Defaults to Date.now().
+         */
+        now?: number
```

Comment:
Should we name this something more descriptive than `now`? Maybe `currentTime` or or `comparisonTime`?